app/src/main/kotlin/de/janhopp/luebeckmensawidget/WidgetConfigurationActivity.kt

```diff
@@ -0,0 +1,37 @@
+package de.janhopp.luebeckmensawidget
+
+import android.appwidget.AppWidgetManager.EXTRA_APPWIDGET_ID
+import android.appwidget.AppWidgetManager.INVALID_APPWIDGET_ID
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import de.janhopp.luebeckmensawidget.ui.config.WidgetConfigurationScreen
+import de.janhopp.luebeckmensawidget.ui.theme.MensaTheme
+
+class WidgetConfigurationActivity : ComponentActivity() {
+    private var appWidgetId = INVALID_APPWIDGET_ID
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
```

Comment:
If you add `enableEdgeToEdge()` here, you might not even need to dig in the xml to change the color of the app bar (if you mean the status bar by app bar). If you really mean the app bar, then I would suggest to simply remove it. xD

Reply:
I meant the system bar 😄 

Gonna fix the system bar then...